config/routes.rb

```diff
@@ -73,6 +73,8 @@
 
   get 'orga/users/info', to: 'orga/users_info#index', as: :orga_users_info
   patch 'orga/seasons/switch_phase', to: 'orga/seasons#switch_phase', as: :switch_phase
+  put 'orga/community/reset_user_availability', to: 'orga/community#reset_user_availability'
```

Comment:
Can you please move this into the `orga` namespaces`?  It will have the `/orga` prefix automatically.